GLAA.Services/PublicRegister/PublicRegisterViewModelBuilder.cs

```diff
@@ -108,7 +108,9 @@ public PublicRegisterLicenceListViewModel BuildEmptySearch()
             {
                 Licences = licences.Distinct().Select(BuildSummary),
                 PublicRegisterSearchCriteria = publicRegisterSearchCriteria,
-                AvailableCountries = BuildAvailableCountries()
+                AvailableCountries = BuildAvailableCountries(),
```

Comment:
Yeah I saw this issue in the admin pages too. I've used @MorayM's `INeedCountry /County` approach and left the mapping generic so you might be able to do that.

Chalk it up and tech debt for the time being